travel/Html/base.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="Boocking system">
+		<meta name="keywords" content="Travel,travels,boocking,tickets,hotels">
+		<title>Travel platform</title>
+	<link href="https://fonts.googleapis.com/css?family=Acme|Wendy+One" rel="stylesheet">	
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
+	
+<body class="container smallscr">
+	{% block content %}
```

Comment:
Please fix formatting for `content` block. Something is wrong with spaces around.